src/plugins/ffe/ffe_event_controller.py

```diff
@@ -53,3 +59,42 @@ async def htmx_ffe_event_tab(
             request,
             event_uniq_id=event_uniq_id,
         )
+    
+    @post(
+        path='/ffe/test-password',
+        name='ffe-admin-tournament-test-auth',
+    )
+    async def htmx_admin_timer_clone(
```

Comment:
Path, name and function name not matching
```suggestion
    @post(
        path='/admin/ffe/test-auth',
        name='admin-ffe-test-auth',
    )
    async def htmx_admin_ffe_test_auth(
```